spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../../lib/takeaway.rb'
+require 'twilio-ruby'
+
+describe "FEATURE SPEC" do
+
+  describe 'takeaway features' do
+
+    it 'US1 - see a list of dishes with prices' do
+      takeaway = Takeaway.new(Menu.new)
+      expect(takeaway.show_menu).to eq ({ribs: 3,
+        beef: 4,
+        rolls: 3,
+        chips: 2,
+        pies: 5})
+    end
+
+    it 'US2 - selects some dishes' do
+      takeaway = Takeaway.new(Menu.new)
+      expect(takeaway.select({beef: 2, rolls: 3}, 17)).to eq ({beef: 2, rolls: 3})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.